kratos/includes/constitutive_law.h

```diff
@@ -821,7 +821,7 @@ class KRATOS_API(KRATOS_CORE) ConstitutiveLaw : public Flags
      * @param rShapeFunctionsValues the shape functions values in the current integration point
      * @param the current ProcessInfo instance
      */
-    KRATOS_DEPRECATED_MESSAGE("Please do not use this method - Use InitializeMaterialResponse instead\"")
+    KRATOS_DEPRECATED_MESSAGE("Please do not use this method\"")
     virtual void InitializeSolutionStep(const Properties& rMaterialProperties,
```

Comment:
Why did you change this message? Maybe you missed this change?

Reply:
OK, I have modified them again and put them as in the master. I do not know why the modified after merging with the master branch...